lib/utils/cloud.ts

```diff
@@ -280,3 +272,9 @@ export async function getFormattedOsVersions(
 	}
 	return versions;
 }
+
+export function formatOsVersion(osVersion: SDK.OsVersion): string {
+	return osVersion.line
+		? `v${osVersion.raw_version} (${osVersion.line})`
+		: `v${osVersion.raw_version}`;
```

Comment:
Before this PR:
```
$ balena os versions raspberrypi4-64
v2.88.4+rev0.prod (recommended)
v2.88.4+rev0.dev
v2.87.16+rev1.prod
...

$ balena os versions --esr raspberrypi4-64
v2021.10.2.prod (next, recommended)
v2021.10.2.dev (next)
v2021.07.1.prod (current)
...
```
After this PR:
```
 $ balena os versions raspberrypi4-64
v2.88.4+rev0.prod
v2.88.4+rev0.dev
v2.87.16+rev1.prod
...

$ balena os versions --esr raspberrypi4-64
v2021.10.2.prod (next)
v2021.10.2.dev (next)
v2021.07.1.prod (current)
...
```

Reply:
A note about this was added to the [v13 release notes](https://github.com/balena-io/balena-cli/wiki/CLI-v13-Release-Notes) as a potentially/arguably breaking change. (This PR was released as version 13.0.1 in less than 24 hours after the release of 13.0.0. Extended [five-second rule](https://en.wikipedia.org/wiki/Five-second_rule) applies?) 😁